webpack.config.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/* eslint-env node */
+
+const path = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+  devtool: 'source-map',
```

Comment:
do we need a source-map because i don't think we are minifying anything?

Reply:
I will look on that, but Babel compile jsx files into js during build process. My guess is that sourcemap is needed for debugging and identify correct files or lines of code.